docs/bluemix-postgres.md

```diff
@@ -133,6 +166,8 @@ Which should result in:
 Welcome to GitLab, <user>!
 ```
 
+> Note: If you are getting the error `Permission denied (publickey).` even though you have added the SSH key, then it might be related to a directory or file permission issue on the gitlab container. This can be due to an ownership problem of the `/var/opt/gitlab/` directory or the ` /var/opt/gitlab/.ssh/authorized_keys` file. Refer to the [stackoverflow question](https://stackoverflow.com/a/42474788) for more details.
```

Comment:
maybe this goes in the Troubleshooting section?  just a though - I think it's actually fine here...